packages/components/switch/src/switch.scss

```diff
@@ -0,0 +1,157 @@
+$sizes: sm, md, lg;
+
+:host {
+  --_toggle-padding: var(--sl-space-input-switch-padding-controller);
+  --_track-radius: calc((var(--handle-size) + var(--_toggle-padding) * 2) / 2);
+
+  --_main-background-color: var(--sl-color-input-switch-default-unchecked-default-background);
+  --_handle-color: var(--sl-color-input-switch-default-unchecked-default-handle);
+  --_icon-color: var(--sl-color-input-switch-default-unchecked-default-icon);
+  --_toggle-container-padding: var(--sl-space-input-switch-padding-container);
+
+  --handle-position: 0%;
+  --isLTR: 1;
+
+  align-items: center;
```

Comment:
Do we want to have variables and other properties for `host` separated like in other components?

Eg.
```
:host {
  --_toggle-padding: var(--sl-space-input-switch-padding-controller);
  --_track-radius: calc((var(--handle-size) + var(--_toggle-padding) * 2) / 2);
  --_main-background-color: var(--sl-color-input-switch-default-unchecked-default-background);
  --_handle-color: var(--sl-color-input-switch-default-unchecked-default-handle);
  --_icon-color: var(--sl-color-input-switch-default-unchecked-default-icon);
  --_toggle-container-padding: var(--sl-space-input-switch-padding-container);
  --handle-position: 0%;
  --isLTR: 1;
  }
  
:host {
  align-items: center;
  display: inline-grid;
  gap: var(--sl-space-input-switch-padding-gap);
  grid-auto-rows: minmax(0, max-content);
  grid-template-areas:
    'inputlabel toggle'
    'hint toggle';
  grid-template-columns: 1fr minmax(0, max-content);
  outline: none;
}
```